pb-rs/src/types.rs

```diff
@@ -781,6 +781,16 @@ fn get_modules(module: &str, imported: bool, desc: &FileDescriptor) -> String {
         .collect()
 }
 
+#[derive(Debug, Clone, Default)]
+pub struct Extend {
+    /// The message being extended.
+    pub name: String,
+    /// All fields that are being added to the extended message.
+    pub fields: Vec<Field>,
+}
+
+impl Extend {}
```

Comment:
I created this extend struct, which hopefully will be incorporated into the compiler eventually (possibly by me), but we may want to switch to #235 first.